content/advanced/330_servicemesh_using_appmesh/integrate_observability/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Observability"
+date: 2020-01-27T08:30:11-07:00
+weight: 90
+draft: false
+---
+
+ In this chapter, we will dive into key operational metrics and tools such as CloudWatch Container Insights, Cloudwatch Logs and Prometheus that you can leverage within the AWS environment to supercharge your ability to monitor metrics, collect logs, trigger alerts, and trace distributed services.
```

Comment:
For the next few modules looking at o11y data, should we have them drive some load? Even if it's a simple while loop curl'ing the frontend, just to have something to look at.

Reply:
We can see if we can add a chapter for this as well